ttnn/cpp/ttnn/operations/sliding_window/reference_sliding_window.cpp

```diff
@@ -12,26 +12,26 @@
 #include "ttnn/tensor/host_buffer/functions.hpp"
 #include "ttnn/tensor/host_buffer/types.hpp"
 
-namespace tt::tt_metal::sliding_window {
+namespace ttnn::operations::sliding_window {
 
 owned_buffer::Buffer<bfloat16> ref_conv_op(
     const Tensor &input_padded_tensor,
     Shape input_nchw_shape,
     uint32_t stride_h,
     uint32_t stride_w,
     const std::vector<float> &filter_vector,
-    Shape &filter_pyt_tensor_shape,
-    Shape &out_golden_pyt_tensor_shape) {
+    Shape filter_pyt_tensor_shape,
```

Comment:
`const Shape&` ?

Reply:
I've removed the reference because when the function is called, I convert tt::tt_metal::Shape to the ttnn::Shape, which becomes an rvalue, and I couldn't pass it as a reference, which needs an lvalue.

Reply:
oh ok